commissioning/app.py

```diff
@@ -0,0 +1,54 @@
+"""Takes configuration information from a file and sets up the Rover."""
+import argparse
+import glob
+import logging
+import os
+from dotenv import dotenv_values
+
+logging.basicConfig()
+LOGGER = logging.getLogger(__name__)
+LOGGER.setLevel(logging.getLevelName('INFO'))
+
+
+def update_env_file(env_source_directory, env_dest_directory):
+    """Copy configuration information from one file to another."""
+    env_dest_path = env_dest_directory + '.env'
+
+    source_files = list(filter(
+        os.path.isfile,
+        glob.glob(env_source_directory + "*.env")))
+    if not source_files:
+        LOGGER.warning(f'No .env files found at {env_source_directory}')
+        return
+    source_files.sort(key=lambda x: os.path.getmtime(x))
+    source_file = source_files[-1]
+    source_values = dotenv_values(dotenv_path=source_file)
+
+    if not os.path.exists(env_dest_directory):
+        os.makedirs(env_dest_directory)
+    if not os.path.exists(env_dest_path):
+        open(env_dest_path, 'a').close()
+    dest_values = dotenv_values(dotenv_path=env_dest_path)
+    dest_values.update(source_values)
+    LOGGER.info(f'Writing .env file with values: {dest_values}')
+    with open(env_dest_path, 'w') as dest_file:
+        for key, value in dest_values.items():
+            dest_file.write(f'{key}={value}\n')
+    LOGGER.info("Done updating .env file.")
+
+
+def main():
+    """Kick off all the commissioning work."""
+    parser = argparse.ArgumentParser()
+    parser.add_argument("source", help="source file directory")
```

Comment:
I think it would be helpful to add validation to these parameters. You could check that the source exists and is readable and that the destination is writable. You can use the [type](https://docs.python.org/3/library/argparse.html#type) parameter to accomplish that

Reply:
> For example, FileType('w') can be used to create a writable file

So it will actually go ahead and create the file?

Reply:
I was just thinking that it can throw an error back to the user if the paths are ultimately going to fail instead of just getting a stack trace

Reply:
Ah, ok.

> type= can take any callable that takes a single string argument and returns the converted value

Do you have any ideas for what that `type=` command would be?

On the source side, I'd like the script to succeed even if the file doesn't exist. In most cases, the Pi will boot with no flashdrive, so the source file won't be present. Right now, the `blob.blob` read gracefully returns an empty list that handles that.